component/admin/views/webservices/view.html.php

```diff
@@ -81,7 +81,19 @@ public function display($tpl = null)
 		$this->return = base64_encode('index.php?option=com_webservices&view=webservices');
 
 		// Check if option is enabled
-		if (JBootstrap::getConfig('enable_webservices', 0) == 0)
+		try
+		{
+			$container = (new Joomla\DI\Container)
```

Comment:
I haven't followed too closely but if this code is supposed to be PHP 5.3 compat then this line fails (it's 5.4+ syntax).